components/purchase_modal/purchase_modal.tsx

```diff
@@ -290,6 +329,73 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
         });
     }
 
+    paymentFooterText = () => {
+        const normalPaymentText = (
+            <FormattedMessage
+                defaultMessage={'Payment begins: {beginDate}'}
+                id={'admin.billing.subscription.paymentBegins'}
+                values={{
+                    beginDate: getNextBillingDate(),
+                }}
+            />
+        );
+
+        let payment = normalPaymentText;
+        if ((!this.props.isFreeTrial && this.state.currentProduct?.billing_scheme === BillingSchemes.FLAT_FEE) &&
+                this.state.selectedProduct?.billing_scheme === BillingSchemes.PER_SEAT) {
```

Comment:
nit: no need for extra parenthesis here
```suggestion
        if (!this.props.isFreeTrial && this.state.currentProduct?.billing_scheme === BillingSchemes.FLAT_FEE &&
                this.state.selectedProduct?.billing_scheme === BillingSchemes.PER_SEAT) {
```